src/as/__tests__/as-pect.d.ts

```diff
@@ -1 +1 @@
-/// <reference path="../../../node_modules/@as-pect/assembly/types/as-pect.d.ts" />
\ No newline at end of file
+/// <reference types="@as-pect/assembly/types/as-pect" />
```

Comment:
@jtenner is this something that needs to be automated in as-pect project generation?